kitsune/products/views.py

```diff
@@ -35,6 +37,16 @@ def product_landing(request, template, slug):
         return HttpResponse(json.dumps({'topics': topic_list}),
                             content_type='application/json')
 
+    if request.LANGUAGE_CODE == settings.WIKI_DEFAULT_LANGUAGE:
+        show_community_support = True
+    else:
+        try:
+            doc = Document.objects.get(locale=settings.WIKI_DEFAULT_LANGUAGE,
+                                       slug='get-community-support')
```

Comment:
Hard Coded Slug is not a good choice after all! Because the slug can change anytime